### ISSUE TITLE
NVDA doesn't announce tabs in IE11

### ISSUE THREAD

**Issue by UNBREAKING** [nvaccess/nvda#9520]
Firstly to say, we are using on project NVDA for checking how accessibility works and found that for our tabs announcing is not working in IE, then I looked into w3 example and found that it is not working also.
### Steps to reproduce: 
1) Open IE 11 and then open: http://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html
2) use navigation to Example of Tab menu
3) use navigation between tabs of Example menu
### Actual behavior:
Announce only of tablist and no announce of any tab.

### Expected behavior:
The same announce as in other browsers such as Google Chrome
#### NVDA installed/portable/running from source:
NVDA installed as app on desktop and turned on
#### NVDA version:
2019.1
#### Windows version:
Windows 10
#### Other information about your system:
Version of IE: 11.1746.15063.0
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

**Comment by Brian1Gaff** [comment]
Is this not due to the lack of current development of IE by Microsoft, they 
kind of just left it. Are you saying it used to work before?
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
-----

**Comment by DrSooom** [comment]
I couldn't reproduce it with IE 11.0.9600.19301 (32-bit) on Win7x64 and NVDA 2018.1. The names of the tabs are announced correctly in focus mode during switching them by pressing ArrowRight and ArrowLeft.

**Comment by Brian1Gaff** [comment]
The obvious  difference is the version of nvda, and windows used between the 
reporter and yourself. I do not know enough about IE version labelling to 
know if it matters on that front.
 It needs another windows 10 IE user to comment I think, but I do stand by 
what I observed earlier about the priority of this problem given such a wide 
choice of better browsers now.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
-----

**Comment by JulienCochuyt** [comment]
> I couldn't reproduce it with IE 11.0.9600.19301 (32-bit) on Win7x64 and NVDA 2018.1.

Tab headers in IE 11 are broken in focus mode since only NVDA 2018.4

> I do stand by
what I observed earlier about the priority of this problem given such a wide
choice of better browsers now.

I am sorry to contradict, but I have to stand against this argument.
Of course there are far better browsers to choose from… when you do have the choice.
Many web applications still in use were designed for IE and only support IE.
This is still really common especially in corporate environments.

For what I could test with latest NVDA 2019.2beta3 / IE 11 / Windows 10, even if PR #9930 gets merged, there will still be an issue under IE with HTML tab headers: Their index and total number is not announced even if "Guess object position information when unavailable" is checked in the "Object Presentation" settings.
